supply/lib/supply/client.rb

```diff
@@ -279,13 +279,13 @@ def latest_version(track)
     def update_listing_for_language(language: nil, title: nil, short_description: nil, full_description: nil, video: nil)
       ensure_active_edit!
 
-      listing = AndroidPublisher::Listing.new(**{
```

Comment:
Since other places already use keyword-ish style, I made this consistent with others.  `**` would make sense where `Hash` object is given from somewhere else whereas this is just a hash literal that we can just omit `{}`.

```
[1] pry(main)> def hello(**args)
[1] pry(main)*   p args
[1] pry(main)* end
=> :hello
[2] pry(main)> hello(a: 1, b: 2)
{:a=>1, :b=>2}
=> {:a=>1, :b=>2}
[3] pry(main)> hello(**{a: 1, b: 2})
{:a=>1, :b=>2}
=> {:a=>1, :b=>2}
[4] pry(main)> hello({a: 1, b: 2})
ArgumentError: wrong number of arguments (given 1, expected 0)
from (pry):1:in `hello'
```

Both result in the same

Reply:
https://github.com/googleapis/google-api-ruby-client/blob/master/google-api-client/generated/google/apis/androidpublisher_v3/classes.rb#L1917-L1919